src/run.py

```diff
@@ -38,15 +38,17 @@ def input_data(file):
 
 if __name__ == "__main__":
     from src.digitaltwin import insert_api_to_table
+    from src.digitaltwin import setup_environment
+    engine = setup_environment.get_database()
     load_dotenv()
     Stats_NZ_KEY = os.getenv('KEY')
```

Comment:
There could be a better descriptive name for the stats nz key than 'KEY'

Reply:
Good point. Already fixed by @caseyli92 :)